build.sbt

```diff
@@ -6,37 +6,34 @@ val scala213 = "2.13.8"
 val scala3   = "3.1.1"
 val allScala = Seq(scala212, scala213, scala3)
 
-val akkaVersion               = "2.6.18"
-val catsEffect2Version        = "2.5.4"
+val akkaVersion               = "2.6.19"
 val catsEffect3Version        = "3.3.12"
 val catsMtlVersion            = "1.2.1"
 val circeVersion              = "0.14.2"
-val http4sVersion             = "0.23.11"
+val http4sVersion             = "0.23.12"
 val laminextVersion           = "0.14.3"
 val magnoliaVersion           = "0.17.0"
 val mercatorVersion           = "0.2.1"
-val playVersion               = "2.8.15"
+val playVersion               = "2.8.16"
 val playJsonVersion           = "2.9.2"
-val sttpVersion               = "3.5.0"
-val tapirVersion              = "0.20.2"
-val zioVersion                = "2.0.0-RC2"
-val zioInteropCats2Version    = "2.5.1.0+11-d4b128a4-SNAPSHOT"
-val zioInteropCats3Version    = "3.3.0-RC2"
-val zioInteropReactiveVersion = "2.0.0-RC3"
-val zioConfigVersion          = "3.0.0-RC2"
-val zqueryVersion             = "0.3.0-RC2"
-val zioJsonVersion            = "0.3.0-RC3"
-val zioHttpVersion            = "2.0.0-RC4"
+val sttpVersion               = "3.6.2"
+val tapirVersion              = "1.0.0-RC3"
```

Comment:
While you're at it, can you use 1.0.0 that was just released?

Reply:
Done